x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -144,7 +148,18 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
       mount: async (params: AppMountParameters) => {
         const [coreStart, startPlugins] = await core.getStartServices();
         const subPlugins = await this.startSubPlugins(this.storage, coreStart, startPlugins);
-        const { renderApp } = await this.lazyApplicationDependencies();
+        const { renderApp, pluginServices } = await this.lazyApplicationDependencies();
+        const appUpdater = new BehaviorSubject<AppUpdater>(() => ({}));
+
+        pluginServices.setRegistry(
+          pluginServiceRegistry.start({
+            coreStart,
+            startPlugins,
+            appUpdater,
```

Comment:
shouldn't we use the same `this.appUpdater$` for the pluginService? I am not sure what it is used for here 🤔